bin/generate.py

```diff
@@ -1275,6 +1275,20 @@ def add_known(obj):
         num_numbered_testcases = 0
         testcase_id = 0
 
+        def parse_count(yaml, warning=False):
+            if yaml is None or 'count' not in yaml or not isinstance(yaml['count'], int):
```

Comment:
Do we want to warn if `count` is given, but not an `int`?

Reply:
this happens inside the TestcaseRule